packages/flutter/lib/src/painting/continuous_rectangle_border.dart

```diff
@@ -76,84 +227,18 @@ class ContinuousRectangleBorder extends ShapeBorder {
     if (b is ContinuousRectangleBorder) {
       return ContinuousRectangleBorder(
         side: BorderSide.lerp(side, b.side, t),
-        borderRadius: BorderRadiusGeometry.lerp(borderRadius, b.borderRadius, t),
+        borderRadius: ui.lerpDouble(borderRadius, b.borderRadius, t),
       );
     }
     return super.lerpTo(b, t);
   }
 
-  double _clampToShortest(RRect rrect, double value) {
-    return value > rrect.shortestSide ? rrect.shortestSide : value;
-  }
-
-  Path _getPath(RRect rrect) {
-    final double left = rrect.left;
-    final double right = rrect.right;
-    final double top = rrect.top;
-    final double bottom = rrect.bottom;
-    //  Radii will be clamped to the value of the shortest side
-    /// of [rrect] to avoid strange tie-fighter shapes.
-    final double tlRadiusX =
-      math.max(0.0, _clampToShortest(rrect, rrect.tlRadiusX));
-    final double tlRadiusY =
-      math.max(0.0, _clampToShortest(rrect, rrect.tlRadiusY));
-    final double trRadiusX =
-      math.max(0.0, _clampToShortest(rrect, rrect.trRadiusX));
-    final double trRadiusY =
-      math.max(0.0, _clampToShortest(rrect, rrect.trRadiusY));
-    final double blRadiusX =
-      math.max(0.0, _clampToShortest(rrect, rrect.blRadiusX));
-    final double blRadiusY =
-      math.max(0.0, _clampToShortest(rrect, rrect.blRadiusY));
-    final double brRadiusX =
-      math.max(0.0, _clampToShortest(rrect, rrect.brRadiusX));
-    final double brRadiusY =
-      math.max(0.0, _clampToShortest(rrect, rrect.brRadiusY));
-
-    return Path()
-      ..moveTo(left, top + tlRadiusX)
-      ..cubicTo(left, top, left, top, left + tlRadiusY, top)
-      ..lineTo(right - trRadiusX, top)
-      ..cubicTo(right, top, right, top, right, top + trRadiusY)
-      ..lineTo(right, bottom - blRadiusX)
-      ..cubicTo(right, bottom, right, bottom, right - blRadiusY, bottom)
-      ..lineTo(left + brRadiusX, bottom)
-      ..cubicTo(left, bottom, left, bottom, left, bottom - brRadiusY)
-      ..close();
-  }
-
-  @override
-  Path getInnerPath(Rect rect, { TextDirection textDirection }) {
-    return _getPath(borderRadius.resolve(textDirection).toRRect(rect).deflate(side.width));
-  }
-
-  @override
-  Path getOuterPath(Rect rect, { TextDirection textDirection }) {
-    return _getPath(borderRadius.resolve(textDirection).toRRect(rect));
-  }
-
-  @override
-  void paint(Canvas canvas, Rect rect, { TextDirection textDirection }) {
-    if (rect.isEmpty)
-      return;
-    switch (side.style) {
-      case BorderStyle.none:
-      break;
-      case BorderStyle.solid:
-        final Path path = getOuterPath(rect, textDirection: textDirection);
-        final Paint paint = side.toPaint();
-        canvas.drawPath(path, paint);
-        break;
-    }
-  }
-
   @override
-  bool operator ==(dynamic other) {
+  bool operator == (dynamic other) {
```

Comment:
== is the "function name" and (dynamic other) is the function's parameter. 

Think of it as `equals(dynamic other)`, which shouldn't be `equals (dynamic other)`.